### PR TITLE
Add test for import image

### LINKED ISSUE #1377: Add Test for Import Image Module
### Please describe the problem (or idea)
This is a sub-issue of #1375 

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by blurry-x-face**
Fixes #1377 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by blurry-x-face** [comment]
@publiclab/is-reviewers please review this

File: test/core/modules/import-image.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const benchmark = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABlBMVEX+AAD///+KQee0AAAAAWJLR0QB/wIt3gAAAAd0SU1FB+EGHRIVAvrm6EMAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMDYtMjlUMTg6MjE6MDIrMDI6MDDGD83DAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTA2LTI5VDE4OjIxOjAyKzAyOjAwt1J1fwAAAABJRU5ErkJggg==',
+  testModule = require('../templates/module-test');
+
+const path = require('path');
+
+const url = path.join(__dirname, '../images/red.png');
+console.log(url);
```
Comment (HarshKhandeparkar) [inline test/core/modules/import-image.js]: Please remove the log.

Comment (blurry-x-face) [reply test/core/modules/import-image.js]: Ohh sorry I forgot, I'll do that ASAP!

[the remark this answers is not in the document]
Comment (blurry-x-face) [reply test/core/modules/import-image.js]: I forgot to remove the log :sweat_smile: , I'll do that ASAP!

**Comment by blurry-x-face** [comment]
> Also, what has changed in the red.png file?

Nothing, i just saved it in the test directory to easily import the image, should I remove that and instead import from example?

**Comment by blurry-x-face** [comment]
@HarshKhandeparkar @debck I've made the requested changes, please re-review this!

**Comment by HarshKhandeparkar** [comment]
No. Not an issue.

**Comment by jywarren** [comment]
Awesome!!!!

**Comment by jywarren** [comment]
This is super. Thank you!

**Pull request merged**